packages/components/charts/src/chart-tooltip.tsx

```diff
@@ -6,9 +6,9 @@ import { useTooltip } from "./use-chart"
 export interface ChartTooltipProps {
   payload: Dict[] | undefined
   dataKey?: string
-  isRadialChart?: boolean
   label?: string
   labelFormatter?: (label: string) => string
+  radialChart?: boolean
```

Comment:
Please don’t remove the `isRadialChart` prop; instead, mark it as `@deprecated`.

Reply:
Fixed.
https://github.com/yamada-ui/yamada-ui/pull/3609/commits/23494200b6df62a54bf9b81703e3843f1c19b25a

Reply:
I reversed the change since it turned out to be a component used internally.
How about this one?

https://github.com/yamada-ui/yamada-ui/pull/3609/commits/7a2a5ac8aa6872360717660cec53e48a35a90f7d

Reply:
@kaehehehe 
I reversed the change since it turned out to be a component used internally.
How about this one?

https://github.com/yamada-ui/yamada-ui/pull/3609/commits/7a2a5ac8aa6872360717660cec53e48a35a90f7d

Reply:
@kaehehehe 
Could you please review this?

Reply:
I'm sorry for the late reply!
LGTM